profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -252,15 +265,107 @@ function nexteuropa_multilingual_form_workbench_moderation_node_unpublish_form_a
 }
 
 /**
+ * Updates fields of URL alias tab in the node edit form.
+ */
+function _nexteuropa_multilingual_alias_node_form_alter(&$form, &$form_state) {
+  if (isset($form['path'])) {
+    $node = $form_state['node'];
+    list($nid, , $bundle) = entity_extract_ids('node', $node);
+    $set_translation_mechanism = variable_get('language_content_type_' . $bundle, 0);
+    // First we need which translation mechanism is used because
+    // the logic to implement is only for entity translation.
+    if ($set_translation_mechanism == ENTITY_TRANSLATION_ENABLED) {
+      // 1. If pathauto deactivated, defining the right default value.
+      // reproduction of the default behavior but with the LANGUAGE_NONE alias.
+      if ($nid) {
+        $und_alias_info = _nexteuropa_multilingual_get_und_entity_alias_info($node, 'node');
+        if (!$und_alias_info['pathauto']) {
+          $form['path']['alias']['#default_value'] = $und_alias_info['alias'];
+        }
+      }
+      // 2. Add new warning message to URL alias field.
+      $custom_alias_description = $form['path']['alias']['#description'];
+      $message = t('This URL alias will be used for ALL translations of this content.');
+      $message_attributes = array(
+        'class' => array('messages', 'warning'),
+        'role'  => array('alert'),
+        'id'    => array('warning_alias_message'),
+      );
+      $variables = array(
+        'message' => $message,
+        'message_attributes' => $message_attributes,
+      );
+      $custom_alias_description .= theme('nexteuropa_multilingual_message', $variables);
```

Comment:
Thanks for introducing a theme function here, it's a nice improvement! Could we move maybe the setting of `$message_attributes` in a theme preprocess function `nexteuropa_multilingual_preprocess_ nexteuropa_multilingual_message(&$variables) ? So that we make the code more readable and we also standardize CSS classes between this call and the one in `_nexteuropa_multilingual_warning_message()`, maybe better if we use only those specified in  `_nexteuropa_multilingual_warning_message()` so we are backward compatible.